docs/source/technical_overview.rst

```diff
@@ -346,27 +346,38 @@ This approach has several advantages over an inheritance-based model:
 The Reduction Process
 ---------------------
 
-The reduction process in pyMOR is handled by so called :mod:`~pymor.reductors`
-which take arbitrary |Models| and additional data (e.g. the reduced
-basis) to create reduced |Models|. If proper offline/online
-decomposition is achieved by the reductor, the reduced |Model| will
-not store any high-dimensional data. Note that there is no inherent distinction
-between low- and high-dimensional |Models| in pyMOR. The only
+The reduction process in pyMOR is handled by so called :mod:`~pymor.reductors`,
+where each reductor takes arbitrary |Models| of associated type, and additional
+data (e.g. the reduced basis) to create reduced |Models|. For instance, the
+:class:`~pymor.reductors.basic.StationaryRBReductor` and
+:class:`~pymor.reductors.basic.InstationaryRBReductor` reductors can reduce any
+:class:`~pymor.models.basic.StationaryModel` and :class:`~pymor.models.basic.InstationaryModel`,
+while the :class:`~pymor.reductors.bt.BTReductor` reduces
+:class:`~pymor.models.iosys.LTIModel` (just to name a few).
+
+If proper offline/online decomposition is achieved by the reductor, the reduced
+|Model| will not store any high-dimensional data. Note that there is no inherent
+distinction between low- and high-dimensional |Models| in pyMOR. The only
 difference lies in the different types of operators, the |Model|
 contains.
 
-This observation is particularly apparent in the case of the classical
-reduced basis method: the operators and functionals of a given discrete problem
-are projected onto the reduced basis space whereas the structure of the problem
+In particular, in most projection-based MOR methods, only the
+operators and functionals of a given discrete problem are projected onto the
```

Comment:
I would be for removing "functionals". They don't appear in sys-mor models, and all functionals are operators. Maybe replacing "operators and functionals" with "|Operators|" is even better.

I would also be for replacing "discrete problem" with "|Model|", since there are models which are not discretizations of a PDE.

Reply:
Done.

Reply:
both sound reasonable to me